src/rpc/rawtransaction.cpp

```diff
@@ -1023,6 +1023,131 @@ static RPCHelpMan testmempoolaccept()
     };
 }
 
+static RPCHelpMan submitrawpackage()
```

Comment:
I have no clue about details of packages and related PRs. Interested in this RPC which can be helpful in my tests. Is this like broadcasting transactions in batches?

```C++
for (int i = 0; i < MAX; i++) {
  if (consensus and mempool policies) {
  // add to broadcast queue
} else {
  // Error
}
}

//BroadcastTransaction(node, queue);

```

Is it possible to select peer for broadcasting this package? Not sure if enough people will ever agree to add such option in `sendrawtransaction` but we can experiment with test RPCs if its easy to implement. Context: https://github.com/bitcoin/bitcoin/issues/21876

So RPC can have one argument to mention peer address.

Reply:
> Is this like broadcasting transactions in batches?

No, but you can make multiple calls to `sendrawtransaction`

> Is it possible to select peer for broadcasting this package?

No, I don't think that would be very helpful here. If you're interested in enabling specific functionality, perhaps open a PR?

Reply:
Thanks